labcodes/labstorybook
# Icons and colors updates

- Adds ability to change Icons sizes
- Refactor color maps and utility classes for color and background-colors
- Change the way we use icons' svgs from background-image to mask-image

**How to test**
Run the project and check if Icons styles are correct (compare with Figma components)
